doc/ref_impl/cntt-ri2/chapters/chapter03.md

```diff
@@ -14,12 +14,13 @@
 <a name="3.1"></a>
 ## 3.1 Introduction
 
-The purpose of this chapter is to list and describe the requirements on labs used to deploy and run the RI2. Overall, the RI2 serves two purposes which guide the definition of requirements in this chapter:
+The purpose of this chapter is to list and describe the requirements on labs used to deploy and run the RI2. Overall, the RI2 serves the following purposes which guide the definition of requirements in this chapter:
```

Comment:
nit
```suggestion
The purpose of this chapter is to list and describe the requirements on labs used to deploy and run the RI2. Overall, the RI2 serves the following purpose which guide the definition of requirements in this chapter:
```

Reply:
Shouldn't it be kept plural as there are three bullets below it?

Reply:
I have to admit that both, singular and plural, sound a bit weird to me - but I am not a native speaker. Just as an alternative proposal, how about something like "... Overall, the RI2 has the following objectives which guide..."?

Reply:
@michaelspedersen yes -- sorry my mistake.